tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -298,3 +300,28 @@ def test_pv_provisioning_under_degraded_state_stop_rook_operator_pod_node(
 
         # Checking cluster and Ceph health
         self.sanity_helpers.health_check()
+
+    @bugzilla('1873938')
+    @skipif_no_lso
+    def test_pv_after_reboot_node(self, nodes):
+        """
+        Verify unexpected PV is not created after node reboot on LSO cluster
+
+        """
+        pv_before_reset = get_pv_names()
+        worker_nodes = get_typed_nodes(
+            node_type=constants.WORKER_MACHINE, num_of_nodes=3
+        )
+        ocp_obj = OCP(kind=constants.PV)
+        for worker_node in worker_nodes:
+            nodes.restart_nodes(nodes=[worker_node], wait=True)
```

Comment:
after node restart, please check all OCS pods status and verify that all the storage pods are in running/completed state.
You can use wait_for_storage_pods() in pods.py